SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
```

Comment:
Maybe we can add a warning here to make it clear right away to not report them on GitHub issues?

Example: https://github.com/nextcloud/server/blob/master/SECURITY.md

```suggestion

> ![WARNING]
> Please do **NOT** report security vulnerabilities through public GitHub issues.

```